tests/test_unittest/test_gtc/test_oir_to_npir.py

```diff
@@ -0,0 +1,119 @@
+from gt4py.gtc import common, oir
+from gt4py.gtc.python import npir
+from gt4py.gtc.python.oir_to_npir import OirToNpir
+
+
+EMPTY_VERTICAL_LOOP = oir.VerticalLoop(
```

Comment:
Maybe just add an builder with only a `build()` (to be extended later if needed)?

Reply:
sure